.github/workflows/ci_workflows.yml

```diff
@@ -73,6 +73,10 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python }}
+    - uses: actions/cache@v2
+      with:
+        path: .hypothesis
```

Comment:
Is this path specification enough? I have no idea where that directory is supposed to be. Is it in some special tox dir, home, root, or ...? How do I find out?

Reply:
I need someone very familiar with tox + hypothesis to chime in here.

Reply:
Tox runs tests in `.tmp/{envname}`, so the `.hypothesis` directory should be created there, but I don't know how to run hypothesis tests locally to check. I guess in actions `{envname}` should always be `${{ matrix.toxenv }}`, so it should work with something like `path: .tmp/${{ matrix.toxenv }}/.hypothesis`, but I think that would mean that each env would have its own cache...
If we want to share the cache across envs, we could modify tox's config to run in `.tmp/` instead of `.tmp/{envname}`. Nothing is generated in this directory except `.hypothesis`, so I don't think we need a per-env tmp directory.

Reply:
Does it matter if the path is different as long as key is the same? 🤔

Reply:
I asked myself the same question ;). I think yes, because actions will store and restore this specific path.

Reply:
@saimn AFAICT you're correct here.  Caching per-env would be fine, or you can configure Hypothesis to use a particular path (e.g. the repo root) - see https://github.com/numpy/numpy/blob/91d9bbebc159ada6cccd0e1fcec1926e26b19c6f/numpy/conftest.py#L17-L21.

I kinda like this approach because it means that updating the tox env won't discard the cache, which is going to be nice for local development too.